engine/consensus/matching/engine.go

```diff
@@ -259,8 +253,9 @@ func (e *Engine) onReceipt(originID flow.Identifier, receipt *flow.ExecutionRece
 	// in the case where a child receipt is dropped because it is received before its parent receipt, we will
 	// eventually request the parent receipt with the `requestPending` function
 	err = e.receiptValidator.Validate(receipt)
+
```

Comment:
I already feel that go code is quite verbose. In this particular case, the error check below directly belongs to the `Validate` call above. What is the reason for adding an extra white space here? 

```suggestion
```